### PR TITLE
docs: update go and node version in running locally

### PR DESCRIPTION
**Pull request by yokishava**
Updated Go and Node versions listed in Running Locally.

### CODE REVIEW

File: docs/running-locally.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ Note:
 
 ## Requirements
 
-* [Go 1.18](https://golang.org/dl/)
+* [Go 1.19](https://golang.org/dl/)
```
Comment (alexec) [inline docs/running-locally.md:41]:
```suggestion
* [Go](https://golang.org/dl/)
```

Comment (yokishava) [reply docs/running-locally.md]: fixed it.

File: docs/running-locally.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ Note:
...
 * [Yarn](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
 * [Docker](https://docs.docker.com/get-docker/)
 * [`protoc`](http://google.github.io/proto-lens/installing-protoc.html)
-* [`node` >= 16.15.0](https://nodejs.org/download/release/latest-v16.x/) for running the UI
+* [`node` >= 19](https://nodejs.org/download/release/latest-v19.x/) for running the UI
```
[the remark this answers is not in the document]
Comment (yokishava) [reply docs/running-locally.md]: fixed it.

**Comment by yokishava** [comment]
Can I close this one because the PR of #10615  revert also contains changes to the document?

**Comment by alexec** [comment]
ok.

**Pull request closed without merging**